resources/authgear/templates/__default__/messages/verification_email.mjml

```diff
@@ -19,6 +19,7 @@
         This email is sent to verify {{ .Email }} on {{ template "app.name" }}. To do so, please click the button below.
       </mj-text>
       <mj-button href="{{ .URL }}" width="300px" padding="24px 0">Verify Email</mj-button>
+      <mj-text font-size="14px" color="#555555" font-weight="light" align="left">Alternatively, use this code in the verification page: {{ .Code }}</mj-text>
```

Comment:
`Alternatively use this code in the mobile app: {{ .Code }}`.

Can we add this paragraph to the plaintext email template as well?

Reply:
Can we follow the design of one time password email so that the code is more visually important?

Reply:
This message is already in the plaintext email template, and I copyed it from there.

Reply:
> Can we follow the design of one time password email so that the code is more visually important?

Not sure if the UX is better this way, since there would be two prominent elements in the email: the big button and big text. It may be confusing to user on whether use the code or click the button.

Reply:
OK. Let's keep what we have now.